src/main/java/seedu/bankwithus/BankWithUs.java

```diff
@@ -83,4 +84,14 @@ public static void main(String[] args) {
             return;
         }
     }
+
+    public  void createAccount() {
+        System.out.println("Whats your name?");
+        ui.createScanner();
+        String userName = ui.getNextLine();
```

Comment:
You might want to trim() the userName. Who knows what kind of shenanigans the user can put in. And try to make sure the user does not enter in the `;` character. We dont want the user corrupting our save file